Tests/Service/Storage/MemcacheTest.php

```diff
@@ -57,10 +57,10 @@ public function testLimitRateNoKey()
               ->method('get')
               ->with('foo')
               ->will($this->returnValue(array('limit' => 100, 'calls' => 1, 'reset' => 1234)));
-        $client->expects($this->exactly(1))
-              ->method('cas')
-              ->with(null, 'foo')
-              ->will($this->returnValue(true));
+//        $client->expects($this->exactly(1))
```

Comment:
It would be nice to just remove this code rather than commenting it. when there is a portion of code commented (without an extra comment explaining why is commented), it is never clear if is a mistake or if that code is not needed.